### ISSUE TITLE
task1_ai_ml_engg : EMDR based therapy algorithmic research

### ISSUE THREAD

**Issue by nlauchande** [omdena/ptsd#6]

**Comment by maelfosso** [comment]
Hi,

Could we have a description, please?

**Comment by nlauchande** [comment]
@maelfosso: The description should be created by the team members.

**Comment by Aya-S** [comment]
I watched a couple of videos to understand EMDR more and how the treatment is administered

- Nice intro video: https://youtu.be/659ySOKyoiU
- EMDR demo session: https://youtu.be/L6UvKhLYf7w
 
 In the second vides the therapist conducted the session over several steps:
1. Summon the traumatic event  
2. Do some prep work with breathing and body posture 
3. Assessment and getting some input from patient on what is comfortable for here regarding the distance between them and the speed of the finger movement 
4. Desensitization involving the hand movement
5. Installation of the improved state 
6. Body scan 
7. Closure and reenforcement 
8. Reevaluation of the outcomes of the therapy 

notes:
- The session was interactive with the therapist adjusting the flow of the session according to input from the patient. 
- The hand movement that guides the patients gaze is adjusted according to input from the patient and the distance moved ( from what i could tell from the vides) is larger than a smart phone screen.

**Comment by Jefiaabby** [comment]
I came across an entire textbook on EMDR and a comprehensive approach to its applications. Since it's a lot to read, I haven't covered everything and I am still reading. I would be able to explain all my findings in a while.

Title and author of the book:
Eye movement desensitization and reprocessing (EMDR) therapy scripted protocols and summary sheets. Treating trauma- and stressor-related conditions
By Marilyn Luber

**Comment by Aya-S** [comment]
I googled more resources on virtual EMDR sessions and there is actual a website that offers EMDR sessions but its is a paid service (virtualemdr.com).
This  https://youtu.be/DALbwI7m1vM  links to a video of another virtual 10 minute EMDR session without the accompanied guidance from a person/professional.
skimming through the comments, seems like people are finding it useful. 
This is promising since this video is quite minimalistic with nothing to guide people through the 8 stages of EMDR which means that even an unguided session through a screen can be of help.

**Comment by maelfosso** [comment]
I just came accross the Youtube video share by @Aya-S and I started reading the book shared by @Jefiaabby 
From the video I think it's possible to use `Re-inforcement Learning` join with `Real-Time computer vision`
The data is the main problem. We could work around that problem either using lot of EMDR video if there are or using Wizard-of-Oz simulations method, looking for a way to adapt it in our situation.

**Comment by maelfosso** [comment]
> I googled more resources on virtual EMDR sessions and there is actually a website that offers EMDR sessions but its is a paid service (virtualemdr.com).
> This https://youtu.be/DALbwI7m1vM links to a video of another virtual 10 minute EMDR session without the accompanied guidance from a person/professional.
> skimming through the comments, seems like people are finding it useful.
> This is promising since this video is quite minimalistic with nothing to guide people through the 8 stages of EMDR which means that even an unguided session through a screen can be of help.

Let's supposed from this video that the doctor is an RL agent trained from many videos and able to guide the patient by analyzing his movement, his behavior, the sound he emits through just a camera.

**Comment by Aya-S** [comment]
> > I googled more resources on virtual EMDR sessions and there is actually a website that offers EMDR sessions but its is a paid service (virtualemdr.com).
> > This https://youtu.be/DALbwI7m1vM links to a video of another virtual 10 minute EMDR session without the accompanied guidance from a person/professional.
> > skimming through the comments, seems like people are finding it useful.
> > This is promising since this video is quite minimalistic with nothing to guide people through the 8 stages of EMDR which means that even an unguided session through a screen can be of help.
> 
> Let's supposed from this video that the doctor is an RL agent trained from many videos and able to guide the patient by analyzing his movement, his behavior, the sound he emits through just a camera.

Hey @maelfosso,
Just saw your reply.
your proposal sounds great it will be amazing to mimic a real practitioner/ therapist.
can we try to dumb it down a bit?
instead of shooting for a fully-fudged simulation what can we start with , that would not be optimum but still adds value ?

Can we start with a fixed video accompanied with maybe an eye tracker ? what do you think ?
